### ISSUE TITLE
chronic cough

### ISSUE THREAD

**Issue by TonyRoscioli** [obophenotype/human-phenotype-ontology#7451]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding as HP:0034315
